picard/tagger.py

```diff
@@ -391,7 +391,8 @@ def handle_command_show(self, argstring):
         self.bring_tagger_front()
 
     def handle_command_quit(self, argstring):
-        self.exit()
+        # hotfix, should be replaced by something better imo
+        thread.to_main(self.sighandler)
```

Comment:
I'm not sure about it but at least it works and I can move further with the implementation